modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/index.tsx

```diff
@@ -69,7 +69,7 @@ const Sidebar = () => {
 			className: 'mt-3',
 			element: (
 				<div
-					className={classNames('pt-3', {
+					className={classNames(' testray-sidebar-item ', {
```

Comment:
Remove unnecessary space in (' ')

```suggestion
					className={classNames('testray-sidebar-item', {
```